src/main/groovy/com/getkeepsafe/dexcount/DexCountException.groovy

```diff
@@ -17,6 +17,8 @@
 package com.getkeepsafe.dexcount
 
 final class DexCountException extends RuntimeException {
```

Comment:
warning: [serial] serializable class DexCountException has no definition of serialVersionUID
public class DexDataException extends RuntimeException {
       ^
error: warnings found and -Werror specified
1 error
1 warning